config.yml.template

```diff
@@ -1,8 +1,17 @@
+# Optional keys are commented out with their defaults as values
 mysql:
     user: ''
     password: ''
-    host: 'localhost'
+#    host: 'localhost'
     raise_on_warnings: True
-    port: 3306
+#    port: 3306
+#    lag_interval: 300
+#    lag_duration: 1800
 
-webhook_url: 'https://hooks.slack.com/services/XXXXX/XXXXX/XXXXX'
\ No newline at end of file
+# Available log levels: DEBUG, INFO, WARNING, ERROR, CRITICAL
+#loglevel: debug
+
+# Use special filename 'stderr' to log to console instead of logging to file.
+#logfile: replication.log
```

Comment:
Better to configure log path in my opinion that just log file name. For example a user might want to log to `/var/log`

Reply:
You mean just in the sample config value? Just change ```replication.log``` to ```/var/log/replication.log```?
Some help text additional clarification?
Or something in the feature code/functionality?

Reply:
@mandoz : ping?

Reply:
@mandoz : Any feedback on this? TIA